src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -123,7 +123,7 @@ public void testUserApi() throws Exception {
             new Header[0]);
         Assert.assertEquals(HttpStatus.SC_NOT_FOUND, response.getStatusCode());
         settings = Settings.builder().loadFromSource(response.getBody(), XContentType.JSON).build();
-        Assert.assertEquals(settings.get("message"), "Role 'opendistro_security_hidden' is not found.");
+        Assert.assertEquals(settings.get("message"), "Role 'opendistro_security_hidden' is not available.");
 
         // Associating with reserved role is allowed (for superamdin)
```

Comment:
should we add similar test to verify that non-superadmin cannot add roles if roles-mapping is reserved

Reply:
There is an existing test to check if non-superadmin cannot add reserved rolesmapping -  https://github.com/opendistro-for-elasticsearch/security/blob/master/src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/UserApiTest.java#L561-L566